packages/foreman/foreman-selinux/foreman-selinux.spec

```diff
@@ -44,7 +36,7 @@ URL:            https://theforeman.org
 Source0:        https://downloads.theforeman.org/%{name}/%{name}-%{version}%{?prerelease:-}%{?prerelease}.tar.bz2
 
 BuildRequires:  checkpolicy, selinux-policy-devel, hardlink
-BuildRequires:  policycoreutils >= %{selinux_policycoreutils_ver}
+BuildRequires:  policycoreutils
```

Comment:
This will still be needed. Otherwise the variable is just unused.

Reply:
I've removed the variable, it's gone.

Reply:
I meant that we still need to set the minimum version. Otherwise `selinux_policy_ver` is just unused.

Reply:
Macro `selinux_policy_ver` is used, I have removed `selinux_policycoreutils_ver` which is not needed anymore, we don't support RHEL6.

Reply:
The `selinux_policy_ver` variable isn't magically used and we must be explicit: `BuildRequires:  policycoreutils >= %{selinux_policy_ver}`